exercise.wwwapi/Endpoints/BookEndpoint.cs

```diff
@@ -0,0 +1,75 @@
+﻿using exercise.wwwapi.Models;
+using exercise.wwwapi.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+namespace exercise.wwwapi.Endpoints;
+
+public static class BookEndpoint
+{
+    public static void ConfigureBookEndpoint(this WebApplication app)
+    {
+        var studentGroup = app.MapGroup("books");
+
+        studentGroup.MapPost("/create/", CreateBook);
```

Comment:
don`t need to `create` part in the url

studentGroup.MapPost("/", CreateBook);

POST call
https://localhost:{port}/Book